src/IBM.WatsonDeveloperCloud/Http/WatsonHttpClient.cs

```diff
@@ -65,16 +67,20 @@ public WatsonHttpClient(string baseUri, string userName, string password, HttpCl
             if (baseUri != null)
                 this.BaseClient.BaseAddress = new Uri(baseUri);
             this.Formatters = new MediaTypeFormatterCollection();
+            this.Formatters.JsonFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("application/octet-stream"));
 
             this.WithAuthentication(userName, password);
         }
 
         public IClient WithAuthentication(string userName, string password)
         {
-            string auth = string.Format("{0}:{1}", userName, password);
-            string auth64 = Convert.ToBase64String(Encoding.ASCII.GetBytes(auth));
+            if (!string.IsNullOrEmpty(userName) && !string.IsNullOrEmpty(password))

```

Comment:
Added because of an error I was getting in Visual Recognition because it was sending an empty authorization header since we are using an apikey.